### DISCUSSION TITLE
Web Interface for setting timezone/location?

### DISCUSSION THREAD

**Discussion by twinotter** [arendst/Tasmota#10210] [Ideas]
I don't see a web interface for setting timezone and location information, despite having a web interface for setting timers. Is there a reason this web interface is missing? (Or is it user error and I just didn't spot it?)

Here are the commands that I use on the console to set this:

backlog latitude 43.08; longitude -89.36; TimeDST 0,2,3,1,2,-300; TimeSTD 0,1,11,1,2,-360; Timezone 99; Time 0

Is there space to include a web configuration for most of this if I write it?

**Comment by Jason2866** [comment]
No, it wastes rare flash for a setting needed only 1 time!
Will not be considered

**Reply by twinotter** [reply]
Module type is only set once. Wifi is generally only set once.

Perhaps settings like these could be built into Tasmotizer. That tool works really well!

**Comment by Jason2866** [comment]
@jziolkowski Timezone setup, a setup feature wish for Tasmotizer
